src/main/java/com/course/fleet/management/API/controller/UsersController.java

```diff
@@ -0,0 +1,39 @@
+package com.course.fleet.management.API.controller;
+
+import com.course.fleet.management.API.entity.User;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import net.datafaker.Faker;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@RestController
+@RequestMapping("fleet-management-API/users")
+public class UsersController {
```

Comment:
Esto me faltó comentarlo, pero en spring boot la convención para los controllers es que estos estén en singular.

```suggestion
public class UserController {
```